doc/sphinx/changelog.rst

```diff
@@ -4,6 +4,56 @@
 Changelog
 =========
 
+Changes in cclib-1.6.4
+----------------------
+
+**Features**
+
+* [GSOC2020] New method: bader calculates QTAIM partial charges (Minsik Cho)
```

Comment:
This is a duplicate line with the one below.

Reply:
Done.